app/src/main/java/fr/neamar/kiss/DataHandler.java

```diff
@@ -199,6 +214,46 @@ public void addToHistory(Context context, String id) {
         DBHelper.insertHistory(context, currentQuery, id);
     }
 
+    public void addShortcut(Context context, ShortcutPojo pojo) {
```

Comment:
i think we should keep the datahandler "data-agnostic" and minmize code inside. Especially, this function can be defined in broadcast/InstallShortcutHandler.java since it's the only place where it's used